src/types/comfy.ts

```diff
@@ -11,12 +11,15 @@ import type { SettingParams } from '@/types/settingTypes'
 
 export type Widgets = Record<string, ComfyWidgetConstructor>
 
-export interface AboutPageBadge {
+export interface BaseAboutPageBadge {
   label: string
-  url: string
   icon: string
 }
 
+export type AboutPageBadge =
```

Comment:
We do have extensions like ComfyUI-Manager adding custom badge to the about panel right now. Changing this interface would break their implementation.